### ISSUE TITLE
Change the Cursor to Pointer on Hovering the Card to Improve UI

### ISSUE THREAD

**Issue by abdullah-k18** [divyanshudhruv/Minifolio#177]
When we hover over the cards in the education section and the services section, the cursor remains same and when hover over the text in the card it changes to insert cursor. To improve the UI the cursor should be changed to pointer when hovered over any card.

**Comment by divyanshudhruv** [comment]
Yeah, I also saw that issue but this issue is already in development mode and fixed. You can view the `dev-main` branch for more info
